apps/src/netsim/NetSimLocalClientNode.js

```diff
@@ -110,18 +110,20 @@ NetSimLocalClientNode.inherits(NetSimClientNode);
 /**
  * Static async creation method. See NetSimEntity.create().
  * @param {!NetSimShard} shard
+ * @param {string} displayName
  * @param {!NodeStyleCallback} onComplete - Method that will be given the
  *        created entity, or null if entity creation failed.
  */
-NetSimLocalClientNode.create = function (shard, onComplete) {
-  NetSimEntity.create(NetSimLocalClientNode, shard, function (err, node) {
+NetSimLocalClientNode.create = function (shard, displayName, onComplete) {
+  var templateNode = new NetSimLocalClientNode(shard);
+  templateNode.displayName_ = displayName;
+  templateNode.getTable_().create(templateNode.buildRow_(), function (err, row) {
```

Comment:
Here I've used our faux-private naming convention as a class-level modifier (C-style) and not as an instance-level modifier (Ruby-style).  Since `templateNode` is an instance of `NetSimLocalClientNode` I was okay with this static method on the same class using a private member.

Some languages don't consider this a violation of encapsulation [because the caller already has full knowledge of the callee's implementation](http://stackoverflow.com/a/16357752).

If you think this is just confusing from a readability point of view, maybe I should just name these (getTable_, heartbeat_, onFailedHeartbeat_) like public members?